lib/Test/Nginx/Socket.pm

```diff
@@ -4462,6 +4462,18 @@ One can also add an optional user group separated by spaces, as in
 
     export TEST_NGINX_WORKER_USER='agentzh wheel'
 
+=head2 Special Variables
+
+=head3 TEST_NGINX_RAND_PORT_XXX
+
+Variables like C<TEST_NGINX_RAND_PORT_XXX> are expanded to random and unused
+unprivileged ports numbers to build the configuration of the server,
+where C<XXX> is an integer from 1985 to 65535.
+
+For instance, C<TEST_NGINX_RAND_PORT_1> will be expanded to an integer port,
```

Comment:
“integer port" => "integer port number" (still thinking "integer" is redundant. is there any float-point number port numbers?)

Reply:
ok, I've changed "integer port" to "port number".